drivers/serial/Kconfig

```
@@ -617,4 +617,27 @@ config PSEUDOTERM_TXBUFSIZE
 
 endif # PSEUDOTERM
 
+menuconfig UART_BTH4
+	bool "BT H4 uart pseudo device"
+	default n
+	select MM_CIRCBUF
+	---help---
+		Enable Bluetooth H4 UART Pseudo Device(eg. /dev/ttyHCI)
```

This description is a bit obscure. Until we get proper documentation for BLE we should make it clear what this really does:
```suggestion
		Enable support for Bluetooth H4 UART Pseudo Device(eg. /dev/ttyHCI). This instantiates a serial-like
		interface over an existing bluetooth controller via HCI interface. Useful for external Bluetooth
		stacks working this way instead of the socket based interface.
		
```